flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -754,7 +766,9 @@ static void uavoMSPBridgeTask(void *parameters)
 
 	while (1) {
 		uint8_t b = 0;
-		uint16_t count = PIOS_COM_ReceiveBuffer(msp->com, &b, 1, PIOS_QUEUE_TIMEOUT_MAX);
+
+		uint16_t count = PIOS_COM_ReceiveBuffer(msp->com, &b, 1, 3000);
```

Comment:
Any reason for 3000? It won't do anything other than unblock this task so it can immediately block again AFAICT?